src/app/shared/services/sqlite-query.service.ts

```diff
@@ -6,6 +6,7 @@ import { SqliteService } from '@keira-shared/services/sqlite.service';
 import { ConfigService } from '@keira-shared/services/config.service';
 import { TableRow } from '@keira-types/general';
 import { QueryService } from '@keira-shared/services/query.service';
+import { Lock } from 'app/features/item/item-template/item-preview';
```

Comment:
we should NEVER import things from `features`. If something is meant to be shared, it should be under `shared/`. I'll move it in my PR.